etcd/os-faq.md

```diff
@@ -0,0 +1,45 @@
+# etcd on Container Linux FAQ
+
+Questions often asked when using etcd v3 on CoreOS Container Linux
+
+## What is the current version of etcd?
+
+The version 3 series is the latest edition of the etcd binary and API. The [current release is the latest release in the version 3 series][etcd-latest-version].
+
+## Where do I find etcd v3 in Container Linux?
+
+Container Linux includes a systemd service, [etcd-member.service][etcd-member-service], that knows how to fetch and run etcd v3 in a Linux container. No etcd v3-series binary is directly included in the Container Linux filesystem.
+
+## Why is etcd v2 still part of the Container Linux filesystem?
+
+For reasons of compatibility, previous, deprecated versions of etcd, named `etcd` and `etcd2`, are included in the Container Linux filesystem until they complete their [sunset schedules][etcd3-blog] and are finally removed.
+
+The etcd (v0) binary will be removed from Container Linux in May, 2017.
+
+The etcd2 binary will be included in the OS until June, 2018.
+
+The recommended way to run etcd on Container Linux is to use the etcd-member service.
+
+## When will v3 be included in the OS image?
+
+Version 3-series etcd binaries are packaged in a Linux container and instantiated by the `etcd-member.service` systemd unit.
+
+# Can I curl the v3 api?
```

Comment:
this is not accurate. etcd3 ships with a http gateway for v3 api: see https://github.com/coreos/etcd/blob/master/Documentation/dev-guide/api_grpc_gateway.md